config/field.storage.paragraph.field_description.yml

```diff
@@ -1,4 +1,4 @@
-uuid: 1dee3bb2-50d3-4107-9493-7e9b078af382
+uuid: afd30bfd-49f9-4f24-a677-3a2d4a684dc6
```

Comment:
If you run into any problems with this, this might be worth noting that the UUID needs to be one or the other. We both likely added the field and UUID is different.